java/src/jmri/jmrit/dispatcher/ActivateTrainFrame.java

```diff
@@ -914,6 +914,13 @@ private String getBlockName(Block b) {
         if (b != null) {
             String sName = b.getSystemName();
             String uName = b.getUserName();
+            // Jay Janzen
```

Comment:
I am inclined to reject this unless the comment is removed from the combo box. This introduces an inconsistency in how layout elements are displayed.